Automattic/jetpack
# AI Client: add AI Client icon components

This PR adds an exposes the AI Icons to be shared with other projects. Confirm you see the icons there. They are not being used yet for any external project.

<img width="505" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/58317f7c-563b-4565-8c30-73641e582977">

Fixes #

## Proposed changes:

* Add AI Client icon components

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






Use storybook

* Go to the storybook folder

```
cd ./projects/js-packages/storybook`
```
* Run the Storybook app
```
pnpm run storybook:dev
```

Confirm you see the icons there.

<img width="534" alt="image" src="https://github.com/Automattic/jetpack/assets/77539/b7ce9eea-efae-4357-8d19-8c1009e9e1a2">

Dev link: http://localhost:50240/?path=/story/js-packages-ai-client-icons--default&globals=backgrounds.grid:!true

They are not being used yet for any external project.
